src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -589,42 +606,76 @@ public void testCatchErrorInProcessFilesMethod() throws Exception {
 
     @Test
     public void testExternalConfigurationResourceDoesNotExist() throws Exception {
-        final Checker mockChecker = createMockCheckerWithCacheForModule(DummyFileSetCheck.class);
+        final File cacheFile = temporaryFolder.newFile();
+        final Checker mockChecker = createMockCheckerWithCacheForModule(
+            DummyFileSetCheck.class, cacheFile);
 
         final String pathToEmptyFile = temporaryFolder.newFile("EmptyFile.java").getPath();
         final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
         // We invoke 'verify' twice to invalidate cache
         // and have two identical exceptions which happen on the same line between runs
         final int numberOfRuns = 2;
+        final String[] md5AfterFirstAndSecondRuns = new String[numberOfRuns];
+        final Properties cache = new Properties();
         for (int i = 0; i < numberOfRuns; i++) {
             verify(mockChecker, pathToEmptyFile, expected);
```

Comment:
I cannot figure out how to intersect the exception. I could generate md5 of expected exception manually and assert that actual md5 after first and second run is equal to expected, however, if we change the line of ```verify``` method invocation the md5 will be changed.